### PR TITLE
add two file to prepare target machine

### PR DESCRIPTION
**Pull request by ELabbassalah56**
add two bash script files to automate  the preparation of ROS target machine

### CODE REVIEW

File: tools/scripts/ros_setup/pkg_dependencies/installSystem.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!bin/bash
+
+echo "#########################################################"
+echo "#  prepration machine to install Vision Package         #"
+echo "#            AURTHOR\ ELabbasSalah                      #" 
+echo "#########################################################"
```
Comment (IbrahimHabibV) [inline tools/scripts/ros_setup/pkg_dependencies/installSystem.sh]: - Prepare machine to install VAUV packages, I think preparation machine is not correct
- Please remove your name from the script, this is not totally professorial , you have already been added to the contributes and the commit is under your name, so all your rights are reserved no worries

File: tools/scripts/ros_setup/pkg_dependencies/getRos2.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!bin/bash
+
+echo "#########################################################"
+echo "#  prepration machine to install Vision Package         #"
+echo "#            AURTHOR\ ELabbasSalah                      #" 
+echo "#########################################################"
```
Comment (IbrahimHabibV) [inline tools/scripts/ros_setup/pkg_dependencies/getRos2.sh]: - Prepare machine to install VAUV packages, I think preparation machine is not correct
- Please remove your name from the script, this is not totally professorial , you have already been added to the contributes and the commit is under your name, so all your rights are reserved no worries

**Comment by IbrahimHabibV** [comment]
@ELabbassalah56  please also make the path to the scripts "tools/scripts/ros_setup", no need for package dependencies yet as this scripts only install ros2 and some common dependences not specific package dependencies 
I want to thank you for the automation script, good job 🥇

**Review by IbrahimHabibV — CHANGES_REQUESTED** [review]

File: tools/scripts/ros_setup/pkg_dependencies/installSystem.sh (hunk 1 above)
Comment (ELabbassalah56) [reply tools/scripts/ros_setup/pkg_dependencies/installSystem.sh]: okay i will update it and  pull them

**Comment by ELabbassalah56** [comment]
i updated which you ordered  @IbrahimHabibV

**Review by IbrahimHabibV — APPROVED** [review]

**Pull request merged**